### DISCUSSION TITLE
Project board automation to move closed issues to the done column.

### DISCUSSION THREAD

**Discussion by agerlic** [community/community#4901] [Projects and Issues] [answered]
Hi, when an issue is closed I'd like to move it automatically to the done column 
I used to rely on [automation for project board ](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-automation-for-project-boards)for that purpose, however I don't see how to do it with Projects (beta)
Thanks

**Comment by wulfland** [comment]
And vice versa - if I move an issue to done, I'd expect it to be closed automatically.

**Reply by eblade** [reply]
I came here to suggest just that! Only having it working one of the ways is like trying to teach your family to not use the old light switches in an automated home...

**Comment by azenMatt** [comment]
We are working on this at the moment :) more automation options to come soon!

**Reply by erikr** [reply]
Can’t wait for automation to ship! Thanks for your hard work!

**Reply by amacneil** [reply]
This is very important for us too, and a big regression in the new projects beta.

We can't use Github actions for automation since our board contains issues from multiple projects. So we need the ability for opening a ticket to put it in the "todo" column, and closing a ticket to move it to the "done" column.

**Comment by aevaldas** [comment, marked as answer]
Hi, I've just found out about the "Workflows": 
![image](https://user-images.githubusercontent.com/38099310/135305090-4b4fe2c5-6cc9-4127-b70a-49d6471638cb.png)


You should be able to reach it on project board by clicking (top right corner of the screen):
![image](https://user-images.githubusercontent.com/38099310/135305160-7b3256d8-3888-4f25-aa45-0b71fba49e85.png)

**Reply by erikr** [reply]
@aevaldas Thanks so much, this solves my problem! This should be marked as the answer.

**Reply by amacneil** [reply]
Oh nice! Is this a new thing? I didn't see it there before.

**Reply by erikr** [reply]
Yeah same, I think it must be new. I kind of wish GitHub was louder about their feature releases, especially to us beta users.

**Reply by arohland** [reply]
They wrote about it here. I recommend following the blog through an RSS reader to get updated about releases. 
https://github.blog/changelog/2021-09-29-the-new-github-issues-09-29-update/

**Reply by joeyfigaro** [reply]
> Hi, I've just found out about the "Workflows": ![image](https://user-images.githubusercontent.com/38099310/135305090-4b4fe2c5-6cc9-4127-b70a-49d6471638cb.png)
> 
> You should be able to reach it on project board by clicking (top right corner of the screen): ![image](https://user-images.githubusercontent.com/38099310/135305160-7b3256d8-3888-4f25-aa45-0b71fba49e85.png)

This UI seems to have been changed? Now there are a fixed number of workflows available to a project board and you can't edit the triggers. There's a workflow for `item closed` but it only allows you to use ticket closures, and the only actions available are moving the ticket to a specific column. We're looking for the *opposite* – set a ticket as closed when it reaches a specific column on the board.

Current UI as of now:
![image](https://user-images.githubusercontent.com/1386477/219097092-9ecd19a0-ac1b-4601-bc60-3264159e359d.png)

**Comment by ronnyroeller** [comment]
It would also be great if closing the issue would move it to the top of the "Done" column. Right now, it goes somewhere in the middle of the column - making it hard to see what was done when (e.g. to see last finished stories during daily standups).

**Reply by SumithChandran** [reply]
Yes this would be great.

**Comment by michellemorrison** [comment]
If an issue is moved to `Done` within beta Project, is it possible to set up an automation to move it to a different beta Project? Use case is one team works through an issue, but then we want to handover to another team for them to start their own unique process.

**Comment by acquiredevelopment** [comment]
I am looking for move item from project board to Done - should close the issue.

**Reply by DannyDevSe** [reply]
why is this still not implemented its really important

**Reply by Auios** [reply]
Looking for this here. I want people to close issues from the project board by moving things to the Done status. Similar to how it works in Jira.

Best I could find was this GitHub action
https://gist.github.com/arthurpalves/59f019c602b91cb91c310c6387ed9300

**Reply by HugoGresse** [reply]
this seems to only be for the classic board, not the v2
